components/Feature/TopicExplorer/index.js

```diff
@@ -4,20 +4,31 @@ const TopicExplorer = (props) => {
   const [searchTerm, setSearchTerm] = useState('')
   const [searchResults, setSearchResults] = useState([])
 
-  const onSearchChange = event => {
+  const getSearchResults = (SearchTerm) => {
     var results = []
-    const newSearchTerm = event.target.value.toLowerCase();
+    const newSearchTerm = SearchTerm.toLowerCase();
```

Comment:
(if you accept the above, you'll need this too)
```suggestion
    const newSearchTerm = searchTerm.toLowerCase();
```

Reply:
done